### DISCUSSION TITLE
86box tearing problems

### DISCUSSION THREAD

**Discussion by yoshi200216** [86Box/86Box#4906] [Q&A] [answered]
### What happened?

hello the mayority of dos games has tearing on opengl software for example: mega race, wetlands and prince of persia 2.
this happen on mid end computers

### Configuration file

```ini
[General]
vid_renderer = qt_software
host_cpu = 13th Gen Intel(R) Core(TM) i5-13400F
emu_build_num = 6130
uuid = 91471b06-3239-5aad-831a-9ae3cbd58f37
vid_resize = 1
window_remember = 1
video_filter_method = 0
mouse_sensitivity = 0.14
video_gl_vsync = 1

[Machine]
machine = ms4144
cpu_family = i486dx
cpu_speed = 50000000
cpu_multi = 1
cpu_use_dynarec = 1
fpu_softfloat = 0
time_sync = local
fpu_type = internal
mem_size = 10240

[Video]
gfxcard = trio64v2dx_pci

[Input devices]
mouse_type = msserial

[Sound]
fm_driver = nuked
sndcard = sb16

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0

[Storage controllers]
hdc = internal
cassette_mode = load

[S3 Trio64V2/DX PCI]
memory = 4

[Microsoft Serial Mouse]
port = 0
buttons = 2

[Hard disks]
hdd_01_parameters = 63, 16, 1377, 0, ide
hdd_01_fn = juandi
hdd_01_speed = ramdisk
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2hd
cdrom_01_speed = 15
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 0:1
cdrom_01_image_path = C:/Users/juand/Downloads/MegaRace (USA) (En,Fr,De,Es,It) (v1.0.1)/MegaRace (USA) (En,Fr,De,Es,It) (v1.0.1).cue
fdd_01_fn = C:/Users/juand/Downloads/Microsoft Mouse 11.0 (3.5)/disk01.img
cdrom_01_type = 86BOX_CD-ROM_1.00

[Monitor #1]
window_coordinates = 0, 23, 1920, 940
window_maximized = 1

[Monitor #2]
window_coordinates = 0, 0, 0, 0
```

### Operating system

Windows 11

### CPU

Intel® Core™ i5-13400F

### 86Box version

v4.2.1

### Build architecture

Windows - x64 (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

**Comment by TC1995** [comment]
At the moment, you can use the software renderer.

**Comment by yoshi200216** [comment]
but with software render the games still has some tearing issues

**Comment by TC1995** [comment]
What's the speed of the emulated machine?

**Comment by yoshi200216** [comment]
50 mhz

**Comment by TC1995** [comment]
The percentage in the title bar I mean.

**Comment by yoshi200216** [comment]
100 percent

**Comment by TC1995** [comment]
It may be an interlaced display.

**Comment by TC1995** [comment]
Screenshot or video?

**Comment by yoshi200216** [comment]
Video

**Comment by TC1995** [comment]
Then show a video of it tearing.

**Comment by OBattler** [comment]
Try the OpenGL 3.0 renderer.

**Comment by yoshi200216** [comment]
heres a video of mega race tearing
https://youtu.be/ehe6ox0XQSk?si=Ia5iPpYTXOIQ9azi

**Comment by yoshi200216** [comment]
I try opengl 3.0 and also has tearing

**Comment by OBattler** [comment]
Then try the Vulkan renderer.

**Comment by OBattler** [comment]
Also, I looked at the video above and I can't see the tearing.

**Comment by yoshi200216** [comment]
Vulkan also has tearing

**Comment by yoshi200216** [comment]
The problem is the game mega race has some artifacts in the cutscenes

**Comment by OBattler** [comment]
At what minute exactly are they?

**Comment by yoshi200216** [comment]
Minute 2:28

**Comment by yoshi200216** [comment]
The majority of dos games have tearing in computers with Intel core i5 13400f and nvidia rtx 3060 ti

**Comment by yoshi200216** [comment]
Even with vsync

**Comment by OBattler** [comment]
I can't see any tearing there, for some reason.

**Comment by yoshi200216** [comment]
See the video in slow mo

**Comment by OBattler** [comment]
I see now. I'm afraid there's not much I can do about it right now, it would require a complete rewrite of the renderer stack.

**Comment by yoshi200216** [comment]
Please fix the tearing on mega race

**Reply by TC1995** [reply, marked as answer]
Not possible at the moment.
